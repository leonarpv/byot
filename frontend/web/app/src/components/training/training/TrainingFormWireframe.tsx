import * as React from 'react';
import {Grid} from '@material-ui/core';

interface Props {
  name?: React.ReactNode;
  mediaList?: React.ReactNode;
  mediaForm?: React.ReactNode;
  button?: React.ReactNode;
}

export const TrainingFormWireframe: React.FC<Props> = (props: Props) => {
  return (
    <Grid container spacing={2} justify="flex-end">
      <Grid item xs={12}>
        {props.name}
      </Grid>
      <Grid item xs={12}>
        {props.mediaList}
      </Grid>
      <Grid item xs={12}>
        {props.mediaForm}
      </Grid>
      <Grid item>{props.button}</Grid>
    </Grid>
  );
};
