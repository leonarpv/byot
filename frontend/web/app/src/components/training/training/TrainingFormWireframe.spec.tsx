import * as React from 'react';
import {render} from '@testing-library/react';
import {TrainingFormWireframe} from './TrainingFormWireframe';

describe('<TrainingFormWireframe/>', () => {
  it('should render', () => {
    const {container} = render(<TrainingFormWireframe />);
    expect(container).toMatchSnapshot();
  });
});
