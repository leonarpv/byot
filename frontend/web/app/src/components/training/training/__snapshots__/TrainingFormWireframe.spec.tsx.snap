// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrainingFormWireframe/> should render 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-justify-xs-flex-end"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    />
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    />
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    />
    <div
      class="MuiGrid-root MuiGrid-item"
    />
  </div>
</div>
`;
